### ISSUE TITLE
Merge Profiles with different ID on different servers managed by the same instance of the IW4M-Admin

### ISSUE THREAD

**Issue by RipleyBooya** [RaidMax/IW4M-Admin#350]
Hello,

For starter: awesome work, we are using your BOT/software for some years now and it is very much apreciated as it render the game so easy to manage. Kudos :)

The issue we are facing (sorry if this has been already assessed, i couldn't find any other issue or FAQ related to this) :

-We have multiple servers and plan to manage all of them through IW4M-Admin.
-Not all servers are on the same game, therefore each unique player gets several IDs.
-At the moment i only configured two of the server on the Admin tool: One IW4x and one Plutonium T6.
-The bot is working very well on both server and, as much as i'm aware, no wrong configuration involved purely between servers and BOT.
-I tried to activate multiple owners but it doesn't quite meet my requierements of having only one profile per user.
-We are using [IW4MAdmin 2024.8.16.1](https://github.com/RaidMax/IW4M-Admin/releases/tag/2024.8.16.1-prerelease).
-We are hosting both gameservers and Admin Bot via our own PELICAN (former PTERODACTYL) panel.

**TL;DR: is there a way to "merge" different profiles/ID from different server using different UID convention ?**

-in the first screen below, both A and B users are myself but on different games.
-second and third screen are my UIDs on the two games/servers.

![image](https://github.com/user-attachments/assets/797d4d1a-4bdd-4bfb-bea7-ce139e2f12b2)

![image](https://github.com/user-attachments/assets/6bc4a88a-4568-4964-b569-c3d7b36980c1)

![image](https://github.com/user-attachments/assets/fe2092df-f9a8-4688-b586-7b0ab27718fb)

I'm not a dev, but i would gladly help if needed with logs/tests or even create VM/VPS for your debuging needs :)
I'm willing to help by messaging here to eventually guide others with same needs, or in vocal with discord or your favorite VoIP solution.

PS. i redacted my IP, but i don't know if the UIDs are sensitive. If the answer is "yes" i will edit my screenshots/attachements.

Thanks a lot for your work and maybe your help if there is a way to do it :)
